src/main/server.ts

```diff
@@ -7,6 +7,8 @@ import * as https from 'https'
 
 const port: number = parseInt(process.env.PORT, 10) || 4000
 
+const logger = require('@hmcts/nodejs-logging').getLogger('server')
```

Comment:
what's the type of `logger`?

Reply:
no idea.. it's a copy paste job from t1 ;)

Reply:
They are amateurs ;)